backend/coins/btc/transaction_test.go

```diff
@@ -125,3 +142,82 @@ func TestGetFeePerKb(t *testing.T) {
 	}
 
 }
+
+func TestPickChangeAddressSucceeds(t *testing.T) {
+	rootFingerprint := []byte{1, 2, 3, 4}
+	signingConfigurations := signing.Configurations{
+		signing.NewBitcoinConfiguration(signing.ScriptTypeP2WPKHP2SH, rootFingerprint, mustKeypath(t, "m/49'/0'/1'"), test.TstMustXKey("xpub6CUmEcJb7juvpvNs2hKMc9BP1n82ixzUb4jyHUdYzSLmnXru3nb4hhGsfS23WRx8hgJLxMxZ7WcBGzTiYfiANUQZe3TVFghLrxvA2Ls7u4a")),
+	}
+
+	testCases := []struct {
```

Comment:
Isn't the main test cases missing? An account with multiple Taproot UTXOs and non-Taproot UTXOs  (should pick a Taproot change address)

Reply:
added!